### PR TITLE
feat: summary email for multiple addresses

### PR DESCRIPTION
**Pull request by wa0x6e**
This PR show all proposals for all the addresses related to the same email address in the same summary email.

### CODE REVIEW

File: test/unit/helpers/snapshot.test.ts [hunk 1]
```diff
@@ -50,13 +50,13 @@ describe('snapshot', () => {
     });
 
     it.each(expectedShortBody)('truncates the body when necessary', async data => {
-      const results = await testSnapshot.getProposals('', data.length);
+      const results = await testSnapshot.getProposals([''], data.length);
```
Comment (bonustrack) [inline test/unit/helpers/snapshot.test.ts:53]: Why this is empty?

Comment (wa0x6e) [reply test/unit/helpers/snapshot.test.ts:53]: We don't test the params, the returned value from the hub is mocked.

Tests are only testing the formatting of the returned value

Comment (bonustrack) [reply test/unit/helpers/snapshot.test.ts:53]: Can it be `[]` then?

Comment (wa0x6e) [reply test/unit/helpers/snapshot.test.ts:53]: typescript is expecting an array of strings

Comment (bonustrack) [reply test/unit/helpers/snapshot.test.ts:53]: Hum weird, you see an error if you use empty array?

**Comment by bonustrack** [comment]
Got this error when I run the tests for some reason
image

**Comment by wa0x6e** [comment]
> Got this error when I run the tests for some reason image

Even with `yarn test`

**Comment by wa0x6e** [comment]
tests seems to work on github actions 

https://github.com/snapshot-labs/envelop/actions/runs/4467434150/jobs/7846889721?pr=29#step:8:1

**Comment by bonustrack** [comment]
@wa0x6e Nvm it works, it was picking a file I had in local

**Review by bonustrack — APPROVED** [review]
tACK

**Pull request merged**